### DISCUSSION TITLE
"TX Exporter for XUMM" ideas and feature requests

### DISCUSSION THREAD

**Discussion by xrpl365** [xrpl365/tx-exporter#1] [Ideas]
## 👋 Welcome!
  Please drop your ideas for future features below.
  * Share your ideas about future features with us and other users in the community.
  * Ask questions about if a feature maybe on our road map.
  * This tool is open-source; we welcome collaboration with other community members.
  * Remember this tool is for the XRPL community, lets build a better future together 💪.

  To get started, comment below with an introduction of yourself and tell us about your ideas.

**Comment by nyhcguy** [comment]
It would be nice to be able to get the pricing data for the transaction.  I am new to interacting with the XRPL.  I would love to contribute.  Great project, thanks.

**Comment by will2022** [comment]
Does it have the ability of showing destination tags of transactions?

**Reply by WietseWind** [reply]
As destination tags are stored inside the transaction details in the DestinationTag field, that should be possible if included in the exportable fields.
